cogs/modmail.py

```diff
@@ -2186,6 +2187,56 @@ async def isenable(self, ctx):
 
         return await ctx.send(embed=embed)
 
+    @commands.command(name="export")
+    @checks.has_permissions(PermissionLevel.ADMINISTRATOR)
+    async def export_backup(self, ctx, collection_name):
```

Comment:
Instead of exporting a collection, most users doesn't know what even are collections. Could you change it so that it exports everything (into one file) and imports them as well?

Reply:
You mean export config/logs and any plugins into 1 bson file? The issue with that would be the bot wont be able to send the large files.